cli/tests/e2e/rules/extract_rules/python_jupyter_paths_exclude.yaml

```diff
@@ -0,0 +1,29 @@
+rules:
+  - id: extract-jupyter-to-python
+    mode: extract
+    languages:
+    - json
+    rules:
+      exclude:
+        - check-for-data
```

Comment:
one issue is that when we process rules, their id gets rewritten based on their location in semgrep-rules/,
such as lang.ocaml.best-practice.check-for-data
so not sure your exclude/include would then match. Maybe we need like for paths: to allow globs (**.check-for-data-*)

Reply:
I was too lazy to figure out the best way to do glob parsing in OCaml so I just made `check-for-data` mean `**.check-for-data`. I couldn't think of any case where that would cause an issue

Reply:
see libs/glob/Parse_pattern.ml (it's now used to parse Rule.ml paths: include/exclude fields (there's even a Rule.glob intermediate type).
But yes I think it's fine for v0 to do what you did